dspace-rdf/pom.xml

```diff
@@ -71,5 +71,9 @@
             <groupId>commons-lang</groupId>
             <artifactId>commons-lang</artifactId>
         </dependency>
+        <dependency>
+        	<groupId>org.dspace</groupId>
+        	<artifactId>dspace-cris-api</artifactId>
```

Comment:
dspace-rdf needs the cris-api for the DataProviderServlet to provide also cris-entities